### PR TITLE
DSFAAP-520: add check answers page for license flow

### PR DESCRIPTION
**Pull request by joseluisgraa**

### CODE REVIEW

File: src/server/common/model/section/license.test.js [hunk 1]
```diff
@@ -9,16 +12,45 @@ describe('License', () => {
...
+
+    it('should return invalid if any nested object is invalid', () => {
+      const licenseData = {
+        emailAddress: ''
+      }
+
+      const result = License.fromState(licenseData).validate()
+
       expect(result.isValid).toBe(false)
+      expect(result.errors).toHaveProperty('license')
```
Comment (hughfdjackson) [inline src/server/common/model/section/license.test.js:33]: ultimately, we're going to want this to be the key of the question that's erroring: 

```javascript
{
  emailAddress: { text: "...." }
}
```

In more complex sections, that will give us the data we need to know which question route users to.

File: src/server/license/check-answers/controller.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const licenseCheckAnswersGetController = {
+  handler(req, h) {
+    const license = req.yar.get('license') ?? {}
+
+    const emailAddress = EmailAddress.fromState(
+      license?.emailAddress
+    ).validate()
```
Comment (hughfdjackson) [inline src/server/license/check-answers/controller.js:17]: If we make the change as suggested above, this is a great place to use LicenceSection.fromState(req.yar.get('licence')

File: src/server/license/check-answers/index.njk [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  {{ govukSummaryList({
+    classes: "govuk-!-margin-bottom-9",
+    rows: [
+      {
+        key: {
+          text: "What email address would you like the licence sent to?"
+        },
+        value: {
+          text: license.emailAddress
```
Comment (hughfdjackson) [inline src/server/license/check-answers/index.njk:17]: this should be the html() method of the `emailAddress` - the one that's designed for rendering a summary, not necessarily for playing back into an input field.

**Review by hughfdjackson — CHANGES_REQUESTED** [review]

File: src/server/common/model/section/license.test.js (hunk 1 above)
Comment (joseluisgraa) [reply src/server/common/model/section/license.test.js]: Done

File: src/server/license/check-answers/controller.js (hunk 2 above)
Comment (joseluisgraa) [reply src/server/license/check-answers/controller.js]: Done

**Review by hughfdjackson — APPROVED** [review]

**Pull request merged**